tests/types/src/valid.ts

```diff
@@ -525,6 +533,15 @@ shippingAddressElement
     }) => {}
   );
 
+shippingAddressElement.update({
+  fields: {
+    phone: 'never',
+  },
+  validation: {
+    phone: undefined,
```

Comment:
qq: can we maybe have a configuration with `phone.required` for `validation.phone` here?